### ISSUE TITLE
I saw your stream

### ISSUE THREAD

**Issue by patrikgafvert** [tsoding/rendering-video-in-c-with-ffmpeg#1]
I saw your twitch stream, and saw it ended in size of 256kb ffmpeg in the pipe.
And then just google the createpipe 256kb, and then saw that you have to provide the buffer size to the pipe, and if you provide NULL, it defaults to it's default size, i wonder if that is 256kb.

https://learn.microsoft.com/en-us/windows/win32/api/namedpipeapi/nf-namedpipeapi-createpipe#parameters

**Comment by rexim** [comment]
Nah, I don't think this was the problem. Here is what it was 6b5631221ae751622f4375769923e09bf7059c78
